docs/sql-ref-syntax-qry-select-setops.md

```diff
@@ -19,4 +19,125 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+Set operators are used to combine two input relations into a single one. Spark SQL supports three types of set operators:
+- `EXCEPT` and `EXCEPT ALL`
+- `INTERSECT` and `INTERSECT ALL`
+- `UNION` and `UNION ALL`
+
+Note that input relations must have the same number of columns and compatible data types for the respective columns.
+
+### EXCEPT and EXCEPT ALL
+`EXCEPT` and `EXCEPT ALL` return the rows that are found in one relation but not the other. `EXCEPT` takes only distinct rows while `EXCEPT ALL` does not remove duplicates.
```

Comment:
`does not remove duplicates.` -> `does not remove duplicates from the result rows.`?